spread.yaml

```diff
@@ -796,6 +796,26 @@ prepare: |
         apt-get update && apt-get install -y eatmydata
     fi
 
+    if [[ "$SPREAD_SYSTEM" == ubuntu-fips-* ]]; then
+        pro status
+
+        if [[ "$SPREAD_REBOOT" = 0 ]]; then
+            echo "--- enabling FIPS"
+            pro enable fips-updates --assume-yes
+            REBOOT
+        else
+            if ! [ "$(cat /proc/sys/crypto/fips_enabled)" = "1" ]; then
+                echo "FIPS not enabled"
+                exit 1
+            fi
+        fi
+
+        # enable GO FIPS PPA
+        add-apt-repository ppa:ubuntu-toolchain-r/golang-fips
+        # XXX pin priority?
+        apt update
```

Comment:
I think `add-apt-repository` has been doing an update for a while now.

Reply:
I think it's true for 22.04, but not for 20.04 which I'll add at some point too.